packages/console/src/ds-components/CopyToClipboard/index.tsx

```diff
@@ -32,6 +32,7 @@ type Props = {
   size?: 'default' | 'small';
   isWordWrapAllowed?: boolean;
   isFullWidth?: boolean;
+  isLeftMarginIgnored?: boolean;
```

Comment:
Can we do this by overriding the className instead of adding a new property?

Reply:
Updated.